src/Angor/Avalonia/Angor.UI.Model/IProject.cs

```diff
@@ -4,8 +4,8 @@ public interface IProject
 {
     public string Id { get; set; }
     public string Name { get; }
-    public Uri Picture { get; }
     public Uri Banner { get; }
+    public Uri Picture { get; }
```

Comment:
We should have a todo to set it to nullable, we might not get a banner and picture.